### PR TITLE
docs: Readme cleanup

### PR DESCRIPTION
**Pull request by anuraghazra**
**What**: Cleaned up readme's `Result` sections to use \<details\> collapsible

**Why**: Because i think those HTML outputs in readme are too long and can be distracting to look at. 

**How**:

**Checklist**:

- [x] Documentation
- [N/A] Tests
- [ ] Ready to be merged

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,7 +97,10 @@ plugins: [
 https://codepen.io/team/codepen/pen/PNaGbb
 ```
 
-#### Result
+<details>
+  <summary>
+  Output
```
Comment (MichaelDeBoey) [inline README.md]: Can you make this `Result` again and make it bold too please?

Also put it on 1 line please.

**Comment by anuraghazra** [comment]
@MichaelDeBoey added the updates.

**Comment by MichaelDeBoey** [comment]
@all-contributors Please add @anuraghazra for docs

**Comment by anuraghazra** [comment]
Welcome 🎉

**Pull request merged**